### ISSUE TITLE
Is this project basically dead?

### ISSUE THREAD

**Issue by YousufSSyed** [0ui/gif-scrubber#8]

**Comment by notetiene** [comment]
Not updating a project for more than 2 years is basically no indication of being dead.  Sometimes, it's an indication of being mature.

**Comment by 0ui** [comment]
@YousufSSyed Nope. Sorry if I was slow to get to your issue. Working through them now.
